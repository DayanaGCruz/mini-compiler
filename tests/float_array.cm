void main() {
  float samples[3];
  float total;
  int idx;

  samples[0] = 1.25;
  samples[1] = 2.50;
  idx = 2;
  samples[idx] = samples[0] + samples[1];
  total = samples[idx] * 2.0;

  print(samples[idx]);
  print(total);
}
