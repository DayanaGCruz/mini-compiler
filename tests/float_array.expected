3.75000000
7.50000000
